scripts/docker/build_docker.py

```diff
@@ -3,17 +3,44 @@
 import argparse
 import sys
 import os
-import os.path
+import fnmatch
+import json
 import time
 import tempfile
 import shutil
 import pprint
 # noinspection PyPackageRequirements
 from termcolor import colored
 import subprocess
+from typing import Union, Iterable, Mapping, Optional, List, Tuple, Set, Dict
+from types import MappingProxyType
 
 
-this_script_path = os.path.dirname(os.path.abspath(__file__))
+class Paths:
+    this_script_folder = os.path.dirname(os.path.abspath(__file__))
+    gatk_sv_path = os.path.dirname(os.path.dirname(this_script_folder))
+    dev_null = "/dev/null"
+
+
+class ImageDependencies:
+    def __init__(
+            self,
+            git_dependencies: Union[str, Iterable[str]],
```

Comment:
It makes it easier for people to add or edit dependencies, since most of the dependencies are only a single glob. The cost of an if statement in the handling of the class logic seems like a reasonable trade-off to me. The alternative would mean that if someone put in one string and forgot to put it in an array, the code would effectively turn it into an array of single characters and bork out. So it's more error-resistant this way too.

Reply:
I personally prefer simpler signatures and go with different overloads if I need to support multiple argument types, but multiple constructors also do not seem very pythonic.